### DISCUSSION TITLE
Is the installation of `pixi` from `conda-forge` officially supported?

### DISCUSSION THREAD

**Discussion by Yura52** [prefix-dev/pixi#887] [Q&A] [answered]
Hi! I noticed that I could install Pixi from conda-forge, but this installation method is not mentioned in the documentation. Is is officially supported?

**Comment by wolfv** [comment, marked as answer]
Hey @Yura52 yes, definitely! Should work pretty much the same wether you get pixi from Github Releases (through the install script) or through conda-forge.

**Reply by Yura52** [reply]
Thank you for the prompt reply!

P.S. Generally, for any software (not necessarily pixi), is there a way to identify whether a given conda-forge package is official or it is a community effort? For pixi, it seems that I could guess it from the maintainers of the github feedstock, but, say, for [uv](https://github.com/conda-forge/uv-feedstock), I don't know how to infer the answer without asking the developers :) 

**UPDATE:** the creator of `uv` [was added](https://github.com/conda-forge/uv-feedstock/pull/6) as a maintainer to `conda-forge/uv-feedstock`.

**Reply by baszalmstra** [reply]
You can check the contributors on the feedstock. In the case of uv I think its community submitted.
